src/test/java/com/exadel/discountwebapp/location/service/LocationIntegrationTest.java

```diff
@@ -0,0 +1,140 @@
+package com.exadel.discountwebapp.location.service;
+
+import com.exadel.discountwebapp.exception.EntityNotFoundException;
+import com.exadel.discountwebapp.location.entity.Location;
+import com.exadel.discountwebapp.location.repository.LocationRepository;
+import com.exadel.discountwebapp.location.vo.LocationRequestVO;
+import com.exadel.discountwebapp.location.vo.LocationResponseVO;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.test.context.SpringBootTest;
+import org.springframework.test.context.jdbc.Sql;
+import org.springframework.test.context.junit4.SpringRunner;
+
+import java.util.ArrayList;
+import java.util.List;
+
+import static org.junit.jupiter.api.Assertions.*;
+
+@SpringBootTest
+@RunWith(SpringRunner.class)
```

Comment:
@RunWith(SpringRunner.class) - is redundant for junit5 tests

Reply:
Removed